source/hardware/controllers/pioneer_ddj_sb3.rst

```diff
@@ -0,0 +1,153 @@
+   :align: center
+   :width: 50%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (mixer section)
+   :figclass: pretty-figures
+
+   Mixer section (p. 8)
+
+===  =====================================================  ============================================================================================
+No.  Control                                                Function
+===  =====================================================  ============================================================================================
+1    :hwlabel:`TRIM` knob                                   Adjust inidvidual channel output gain
+2    :hwlabel:`HI`/:hwlabel:`MID`/:hwlabel:`LOW` knobs      Adjust high/mid/low-frequencies
+3    :hwlabel:`FILTER` knob                                 Low pass / high pass filter for each channel
+4    :hwlabel:`HEADPHONE CUE 1` button                      Toggle headphone pre-fader listening of left deck.
+4    :hwlabel:`HEADPHONE CUE 2` button                      Toggle headphone pre-fader listening of right deck.
+5    Channel faders                                         Adjust the output level for each channel
+6    Cross fader                                            Fade between left and right deck
+7    :hwlabel:`MASTER` knob                                 Adjust master output level
+8    :hwlabel:`HEADPHONES` knob                             Adjust headphones output level
+9    :hwlabel:`MASTER` button                               Output master to headphones
+===  =====================================================  ============================================================================================
+
+Performance Pads
+~~~~~~~~~~~~~~~~
+
+You can use the Pad Mode Select buttons to select a mode for the
+performance pads. Performance pads are numbered from top left to bottom right. First row is 1-4 and the second row is 5-8.
+
+Hot Cue Mode
+^^^^^^^^^^^^
+
+.. figure:: ../../_static/controllers/pioneer_ddj_sb3_hotcue.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (performance pads)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-SB3 (performance pads)
+
+Bookmark positions in the track and jump to them.
+
+========  ===============================================================  ==========================================
+No.       Control                                                          Function
+========  ===============================================================  ==========================================
+1-4       Pad (unlit)                                                      Save current position as hot cue.
+1-4       Pad (lit)                                                        Jump to hot cue.
+1-4       :hwlabel:`SHIFT` + Pad (lit)                                     Clear hot cue.
+5         Pad                                                              Previous track
```

Comment:
What does this mean?

Personally, I think it would be better if you mapped 8 hotcues instead.

Reply:
That is the default "OOB" mapping, which matches the Pioneer manual. Does it matter if the mapping is not the "original" or is the 8 hotcue more common? I know there's a fork/branch with an alternative 8 hotcue mapping.

Reply:
Okay, if it machte the original mapping then it makes sense to map it the same in Mixxx. If someone needs 8 hotcues, that person can open a PR to make this configurable.

Reply:
These are the original (OOB) mappings for the controller.

Reply:
I still don't think this is a good idea. The out of the box mapping is limited to 4 hotcues because Serato DJ Lite is restricted to that. Mappings should not inherit the design decisions from serato whos only purpose is too "persuade" people into spending money on their Pro version.

Reply:
I agree with @Swiftb0y, imitating other software verbatim is an antipattern, especially copying artificial restrictions. Please change this.